tdtax/supernovae.yaml

```
@@ -66,3 +97,6 @@ subclasses:
   other names: [FELT]
   links:
     - https://en.wikipedia.org/wiki/Fast_blue_optical_transient
+- class: Ca-rich
+  tags: [hydrogen poor]
+  other names: [Ca strong]
```

add a newline at end:
```suggestion
  other names: [Ca strong]
  
```